static/styles/mainpage.css

```diff
@@ -13,6 +13,130 @@ body {
     line-height: 18px;
 }
 
+/* NAVBAR */
```

Comment:
we don't have to do this in this PR, but another suggestion might be to move these sections into different css files, for example, we could have a navbar.css file and a details.css file etc etc